yarn.lock

```diff
@@ -8056,6 +8109,18 @@ glob@^10.2.2, glob@^10.3.10:
     minipass "^5.0.0 || ^6.0.2 || ^7.0.0"
     path-scurry "^1.10.1"
 
+glob@^10.3.12:
```

Comment:
Could you deduplicate yarn.lock? It looks like yarn added a bunch of dupes again.